docs/source/operations/projections/omerc.rst

```diff
@@ -62,7 +148,10 @@ Optional
 
 .. option:: +no_rot
 
-    Do not rotate axis.
+    No rectification (not "no rotation" as one may well assume).
```

Comment:
Consider using a note or warning (`.. note:: this is a note`) for the disclaimer to the `+no_rot` option. This makes it more visible.